### PR TITLE
fix(suite): blur url 2.0

### LINKED ISSUE #13528: Invalid URL gets detected as a link and is blurred in token name
**Describe the bug**
If a user receives an Ethereum token where the token's name includes a string such as `2.0`, Trezor Suite mistakenly detects it as a valid link and incorrectly blurs it for phishing protection.

**Info:**
- Suite Version: desktop 24.7.2 (3282fe8)
- OS: Linux x86_64 

**How to reproduce**

1. Create an Ethereum account
2. Receive Lido Staked Ether (stETH) tokens
3. Navigate to tokens page
4. Observe token name being partially blurred

**Expected behavior**
`2.0` and other invalid URLs should not be detected as links, and should not be blurred in the tokens page.

**Screenshots**
Screenshot from 2024-07-28 15-21-54
Screenshot from 2024-07-28 15-21-44

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- 2.0 was incorrectly marked

## Related Issue

Resolve #13528 

## Screenshots:

unresolved because of special chars
Screenshot 2024-07-29 at 9 34 24

### CODE REVIEW

**Review by karliatto — CHANGES_REQUESTED** [review]
There should be 2 commits in this PR.
The current one with prefix `fix(suite):....`
And second one with changes to `@trezor/utils` with prefix `feat(utils): ...`

The package  `@trezor/utils` is published to NPM and having commits with proper name and separated by package allow us to generate better automated CHANGELOGS when we do the releases.

**Comment by tomasklim** [comment]
@karliatto done

File: packages/utils/src/extractUrlsFromText.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const urlRegex =
```
Comment (AdamSchinzel) [inline packages/utils/src/extractUrlsFromText.ts]: nitpick: maybe it will be better to have it like this:
```suggestion
const URL_REGEX =
```

**Review by AdamSchinzel — APPROVED** [review]
Nice tests

**Review by karliatto — APPROVED** [review]

File: packages/utils/src/extractUrlsFromText.ts (hunk 1 above)
Comment (tomasklim) [reply packages/utils/src/extractUrlsFromText.ts]: done

**Pull request merged**